src/awx_plugins/credentials/plugins.py

```diff
@@ -747,10 +747,29 @@
                 'type': 'boolean',
                 'secret': False,
             },
+            {
+                'id': 'request_timeout',
+                'label': gettext_noop('Verify SSL'),
+                'type': 'string',
+                'secret': False,
+                'default': 'galaxy',
+                'help_text': gettext_noop(
+                    'Specify the timeout Ansible should use in requests to'
```

Comment:
Missing a whitespace in the implicit string concatenation. If you leave it like this, it'll render as `"tohost"` in the place where strings merge.
```suggestion
                    'Specify the timeout Ansible should use in requests to '
```

Reply:
nice catch, thanks @webknjaz 
this is fixed now.

Reply:
@pb82 do you know you can just accept suggestions from GH UI?